index.html

```diff
@@ -1355,6 +1367,199 @@ <h4>Exchange Examples</h4>
+{
+  "protocols": {
+    "website": "https://app.example/redirects/z8j3kfk2lQ",
+    "vc-api": "https://app.example/workflows/123/exchanges/987",
+    "oid4vp": "openid4vp://?client_id=https%3A%2F%2Fapp.example%2Fworkflows%2F123%2Fexchanges%2F987%2Fopenid%2Fclient%2Fauthorization%2Fresponse&request_uri=https%3A%2F%2Fapp.example%2Fworkflows%2F123%2Fexchanges%2F987%2Fopenid%2Fclient%2Fauthorization%2Frequest'"
+  }
+}
+        </pre>
+
+        <p>
+When the [=interaction URL=] is fetched using any unrecognized `Accept` header,
+a `text/html` document MUST be returned with directions instructing a human
+being to use specific software that understands how to process interaction URLs.
+        </p>
+
+        <p class="note" title="Mapping to exchange URLs">
+Some coordinator implementations will implement the protocols endpoint as a pass
+through to a protocols endpoint for an exchange instance. For example, a GET on
+`https://app.example/interactions/z8n38Dp7a?iuv=1` will result in a pass-through
+GET on `https://app.example/workflows/123/instances/987/protocols`, which would
+return the response above. Implementing interaction URLs in this way can
+provide an easier implementation path.
+        </p>
+
+      </section>
+
+      <section>
+        <h4>`website` Interaction Protocol</h4>
+        <p>
+The `website` interaction protocol is used by a local system to signal to the
+remote system that it would like to redirect the remote system to a specific
+URL, such as a website where an individual can engage in a use-case specific
+interaction. If the `website` interaction protocol is selected, the local
+system sends data using an HTTP POST to instruct the remote system where to
+send the individual. An example of the POST data is shown below:
+        </p>
+
+        <pre class="example nohighlight" title="A website interaction protocol redirect request">
+{
+  "url": "https://website.example/transactions/8372974",
+  "purpose": "Checkout at ShopCo",
+  "referenceId": "417bcaf2-14d9-11f0-99d7-9f094678517b"
+}
+        </pre>
+
+      </section>
+
+      <section>
+        <h4>`vc-api` Interaction Protocol</h4>
+        <p>
+The `vc-api` interaction protocol is used to initiate a specific [=exchange=]
+as described in Section [[[#participate-in-an-exchange]]].
+        </p>
+
+        <pre class="example nohighlight" title="A VC-API interaction protocol request">
+{
+  "verifiablePresentationRequest": {
+    "query": [{
+      "type": "QueryByExample",
+      "credentialQuery": [{
+        "reason": "Please provide your student ID.",
+        "example": {
+          "@context": [
+            "https://www.w3.org/ns/credentials/v2",
+            "https://www.w3.org/ns/credentials/examples/v2",
+          ],
+          "type": "StudentIdCredential",
+          "credentialSubject": {
+            "studentId": ""
+          },
+        },
+        "trustedIssuer": [{
+          "required": true,
```

Comment:
IIRC, "required" is not necessary here; if trusted issuers are provided then those are the only ones that will be accepted.

```suggestion
```